db/migrations/20210107172948_residents3.js

```diff
@@ -0,0 +1,26 @@
+exports.up = function (knex) {
```

Comment:
Third time's a charm! Once we get the server configured, should we consider adding a table for Appointments and linking to resident's id?

Reply:
I think this wouldn't be too difficult to achieve 🤡